src/logic_analyzer.cpp

```diff
@@ -894,10 +886,21 @@ bool LogicAnalyzer::exportTabCsv(QString separator, QString filename)
 
 void LogicAnalyzer::btnExportPressed()
 {
+	bool paused = false;
+
 	if( !main_win->session_.is_data())
 		return;
 
+	if(m_running) {
+		paused = true;
+		startStop(false);
+	}
+
 	QString filename = saveToFile();
+
+	if(paused) {
+		startStop(true);
+	}
```

Comment:
curios: what is the style-guide for single-block if statements?
depending on who you ask, some projects recommend not adding curly braces for them
so:
```
if(paused)
    startStop(true);
```

Reply:
We are using curly braces for single-block if statements. But the old code (before the style-guide was put in place) was not cleaned up / updated yet.

Reply:
ack
thanks for the info :)

btw: how much is the fish?